db/migrate/20230126140236_add_author_reference_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorReferenceToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] You have done a great job adding the user's reference to the post table. However, this will result in having a user tag/name and according to the project requirements ERD, the post is supposed to reference the user's table as an author. You could instead do it this way: ```add_reference :posts, :author, foreign_key: {to_table: :users}```

![image](https://user-images.githubusercontent.com/86318284/214901288-b5fe7369-2a4b-4fad-8632-0d16b19bef63.png)